### PR TITLE
Added the ability to open an existing project in GUI

### PR DESCRIPTION
**Pull request by rwmtse**
Implemented blackberry/webworks-gui#15
Not ready to merge yet

@rcruz @bryanhiggins Need open icon

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -39,6 +39,10 @@
             <ul>
                 <li>
                     <a href="#" class="title-item-menu">
+                        Open Project
+                        <div class="open-project tooltipstart" data-page="project_open.html" title="Open existing project"></div>
```
Comment (rcruz) [inline public/index.html]: I dont think we want to activate "open project" with a small button like we do with new project. Instead, let's make the entire "Open project" option be clickable. See views/projectList for how the "Certificates" option works in the side bar

Comment (kwallis) [inline public/index.html]: Hey Rowell. Not sure that I understand what you are proposing here? Are you indicating that there should be a separate line entry in the side bar for open project?

Comment (rcruz) [inline public/index.html]: This pull request currently has this appearance:

screen shot 2014-01-16 at 2 00 53 pm

**Comment by rwmtse** [comment]
@rcruz Fixed menu item and remove project name field

**Comment by rcruz** [comment]
r+

**Pull request closed without merging**